app/routes/circulars/circulars.server.ts

```diff
@@ -63,8 +63,10 @@ export async function list({
 
   // Calculate pagination assuming that last === the number of records.
   const totalPages = Math.ceil(last / limit)
-  const firstIdOnPage = Math.max(last - page * limit + 1, 1)
-
+  const firstIdOnPage = Math.max(
```

Comment:
This change was mainly for an issue locally where the `limit` far exceeds the `last`, and was causing an issue where no circulars would show up

Reply:
OK, would you please split that out as a separate PR?

Reply:
I think you actually solved this in a previous commit, my branch was just out of date